src/styles/ComponentPage.scss

```diff
@@ -50,5 +45,8 @@
 }
 
 .demo-content {
+  width: 1000px;
```

Comment:
Should this be `max-width` like it was for `.demo-wrapper` before?

Reply:
No, the width of its child `demo-wrapper` will resize responsively since width isn't set on that.